docs/starters.yml

```diff
@@ -5022,3 +5022,17 @@
     - Pagination between blog posts
     - Has a "archive" page that categorizes and displays all the blog posts by date
     - Mobile friendly
+- url: https://gatsby-starter-material-emotion.netlify.com
+  repo: https://github.com/liketurbo/gatsby-starter-material-emotion
+  description: Gatsby starter of Material-UI with Emotion 👩‍🎤
+  tags:
+    - Material-UI
```

Comment:
Hi @liketurbo you can change the Material-UI to Material and also please check other categories that you have used. To use appropriate categories, you can find the list [here](https://www.gatsbyjs.org/starters/?v=2).

Reply:
@amandeepmittal I didn't know how to change file without creating new commit, so I recreated my [PR](https://github.com/gatsbyjs/gatsby/pull/20675)